src/test/java/mate.academy/security/AuthenticationServiceImplTest.java

```diff
@@ -0,0 +1,60 @@
+package mate.academy.security;
+
+import java.util.Optional;
+import java.util.Set;
+import mate.academy.exception.AuthenticationException;
+import mate.academy.model.Role;
+import mate.academy.model.User;
+import mate.academy.service.RoleService;
+import mate.academy.service.UserService;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+import org.springframework.security.crypto.password.PasswordEncoder;
+
+class AuthenticationServiceImplTest {
+    private static final String EMAIL = "sobaka@mail.ru";
+    private static final String  PASSWORD = "12345";
+    private AuthenticationService authenticationService;
+    private UserService userService;
+    private RoleService roleService;
+    private PasswordEncoder passwordEncoder;
+    private User user;
+
+    @BeforeEach
```

Comment:
We can init service only once before all tests, since we won't change their inner state